### PR TITLE
[ko] Update outdated files in dev-1.21-ko.3(M28-M36)

### PR DESCRIPTION
**Pull request by seokho-son**
Fix following outdated files (#28079)

1. [ ] M28.  `content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md`  | 7(+XS) 0(-)
1. [ ] M29.  `content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md`  | 1(+XS) 1(-)
1. [ ] M30.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md`  | 1(+XS) 1(-)
1. [ ] M31.  `content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md`  | 1(+XS) 1(-)
1. [ ] M32.  `content/en/docs/tasks/manage-kubernetes-objects/kustomization.md`  | 37(+M) 1(-)
1. [ ] M33.  `content/en/docs/tasks/run-application/delete-stateful-set.md`  | 3(+XS) 3(-)
1. [ ] M34.  `content/en/examples/application/job/redis/worker.py`  | 1(+XS) 1(-)
1. [ ] M35.  `content/en/examples/controllers/daemonset.yaml`  | 1(+XS) 0(-)
1. [ ] M36.  `content/en/includes/task-tutorial-prereqs.md`  | 2(+XS) 2(-)

### CODE REVIEW

File: content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 1]
```diff
@@ -177,7 +176,7 @@ min-kubernetes-server-version: v1.10
```
Comment (yoonian) [inline content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:176]: 이번 변경과는 상관없지만, 126 번째 줄에, 
```
1.   `kubectl port-forward` 명령어는 파드 이름과 같이 리소스 이름을 사용하여 일치하는 파드를 선택해 포트 포워딩하는 것을 허용한다.
여기 1. 다음에 공란 3개를
--> 
1.  `kubectl port-forward` 명령어는 파드 이름과 같이 리소스 이름을 사용하여 일치하는 파드를 선택해 포트 포워딩하는 것을 허용한다.
공란 2개로
```
수정하면 좋겠네요. [웹뷰](https://kubernetes.io/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster/#파드의-포트를-로컬-포트로-포워딩하기)로 보니까, 코드 형식으로 나와서요. 확인은 못해보았습니다만...

**Review by yoonian — COMMENTED** [review]
@seokho-son 이번에 수정된 것은 아니지만, 확인 부탁드려요. 나머진 좋습니다!

**Comment by seokho-son** [comment]
@yoonian 반영 및 커밋 스쿼싱까지 완료하였습니다. 감사합니다! :)

**Comment by seokho-son** [comment]
@gochist PR 검토 및 승인 부탁드립니다..^^

**Pull request merged**